src/esp/gfx/replay/ReplayManager.h

```diff
@@ -37,13 +38,8 @@ class ReplayManager {
    * @brief Set callbacks that are invoked when simulation state changes upon
    * playing a new keyframe. This is required to construct Player instances.
    */
-  void setPlayerCallbacks(
-      const Player::LoadAndCreateRenderAssetInstanceCallback&
-          loadAndCreateRenderAssetInstanceCallback,
-      const Player::ChangeLightSetupCallback& changeLightSetupCallback) {
-    loadAndCreateRenderAssetInstanceCallback_ =
-        loadAndCreateRenderAssetInstanceCallback;
-    changeLightSetupCallback_ = changeLightSetupCallback;
+  void setPlayerCallbacks(const PlayerCallbacks& playerCallbacks) {
+    playerCallbacks_ = playerCallbacks;
```

Comment:
```suggestion
  void setPlayerCallbacks(PlayerCallbacks playerCallbacks) {
    playerCallbacks_ = std::move(playerCallbacks);
```
Do not copy this if you don't need to (ie if it's an rvalue).